.gitignore

```diff
@@ -162,8 +162,9 @@ cython_debug/
 /.pids
 /.pants.workdir.file_lock*
 
+credentials.json
 ### Pyenv ###
 
 .python-version
-
+*.db
```

Comment:
Is `*.db` part of Pyenv ignores? If not create a custom section similar to how it's done in [TrustList](https://github.com/CodeForAfrica/TrustList/blob/f485f0c5330ffb3996b3be2d941d0411505bc08c/.gitignore#L320-L331)